apps/www/src/app/(authenticated)/[username]/[post_id]/comment-dropdown.tsx

```diff
@@ -99,10 +99,15 @@ export default function CommentDropdown({
         <AlertDialog open={openDelete} onOpenChange={setOpenDelete}>
           <AlertDialogContent>
             <AlertDialogHeader>
-              <AlertDialogTitle>Delete Comment</AlertDialogTitle>
+              <AlertDialogTitle>
+                {level === 0 ? "Delete Comment" : "Delete Reply"}
+              </AlertDialogTitle>
               <AlertDialogDescription>
-                This action cannot be undone. This will permanently delete your
-                account and remove your data from our servers.
+                {/* This action cannot be undone. This will permanently delete your
```

Comment:
Remove the comments

Reply:
Done. Please check my latest commit/update to the PR!